### DISCUSSION TITLE
How to filter with itemgroup?

### DISCUSSION THREAD

**Discussion by towry** [mrjones2014/legendary.nvim#389] [Q&A] [answered]
I want to use legendary as folder action select, so I can select folder from telescope and use legendary.

So I write the script as following:

```lua
return function (cwd)
  legendary.funcs({
    itemgroup = 'Folder action',
    funcs = {},
})

legendary.find({
  select_prompt = 'Folder action under: ' .. cwd,
  filters = {
   --- how to write this??
  }
})
end
```

how to write the filters so that only everything under 'Folder action' item group shown?

I have looked over the docs, nothing found.

**Comment by mrjones2014** [comment, marked as answer]
A filter is just a function with the signature `fun(item): bool`, however there is currently no way to _start the finder with an itemgroup already selected._ That will require some plugin changes, but it's doable.
